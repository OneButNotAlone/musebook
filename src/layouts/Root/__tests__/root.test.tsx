import * as React from 'react';
import { render } from 'react-testing-library';
import 'jest-styled-components';

import { Root } from '../';

test('Root layout matches snapshot', () => {
    const { container } = render(<Root>Root Layout wraps around everything</Root>);
    expect(container).toMatchSnapshot();
});
