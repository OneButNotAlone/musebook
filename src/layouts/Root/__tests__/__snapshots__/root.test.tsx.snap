// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root layout matches snapshot 1`] = `
@media screen and (min-width:2000px) {
  .c0:root {
    --max-width: 50vw;
  }
}

@media screen  and (min-width:1200px) and (max-width:2000px) {
  .c0:root {
    --max-width: 70vw;
  }
}

@media screen  and (max-width:1200px) {
  .c0:root {
    --max-width: 90vw;
  }
}

<div>
  <div
    class="c0"
  >
    Root Layout wraps around everything
  </div>
</div>
`;
